python/raft-ann-bench/src/raft-ann-bench/run/conf/datasets.yaml

```diff
@@ -0,0 +1,85 @@
+- name: bigann-100M
```

Comment:
My PR brought in new json configs for wiki-all-88M, wiki-all-10M, and wiki-all-1M. Can you add those here?